typescript/web/heap.ts

```diff
@@ -0,0 +1,19 @@
+let customDestination = {
```

Comment:
Should we call these <destination>DestinationInterface?
```suggestion
let heapDestinationInterface = {
```